From 339a2bc99269466e05a546de2ac63c3cb46a2e6e Mon Sep 17 00:00:00 2001
From: David Gardner <git@icmfp.com>
Date: Tue, 26 Feb 2019 19:11:43 +0000
Subject: [PATCH] Add support for the FS-iA6B receiver

---
 utils/inputattach.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/utils/inputattach.c b/utils/inputattach.c
index 53528551..dd3112d6 100644
--- a/utils/inputattach.c
+++ b/utils/inputattach.c
@@ -864,6 +864,9 @@ static struct input_types input_types[] = {
 { "--wacom_iv",		"-wacom_iv",	"Wacom protocol IV tablet",
 	B9600, CS8 | CRTSCTS,
 	SERIO_WACOM_IV,		0x00,	0x00,	0,	wacom_iv_init },
+{ "--fsia6b",          "-fsia6b",      "FS-iA6B RC Receiver",
+       B115200, CS8,
+       100,            0x00,   0x00,   0,      NULL },
 { "--pulse8-cec",		"-pulse8-cec",	"Pulse Eight HDMI CEC dongle",
 	B9600, CS8,
 	SERIO_PULSE8_CEC,		0x00,	0x00,	0,	NULL },
@@ -978,6 +981,7 @@ int main(int argc, char **argv)
 	case 9600: type->speed = B9600; break;
 	case 19200: type->speed = B19200; break;
 	case 38400: type->speed = B38400; break;
+	case 115200: type->speed = B115200; break;
 	default:
 		fprintf(stderr, "inputattach: invalid baud rate '%d'\n",
 				baud);
-- 
2.11.0

